e2e-tests/tests/infra/StudioPlaywrightPage.ts

```diff
@@ -238,15 +238,10 @@ export default class StudioPlaywrightPage {
     const images = await this.preview.getByRole("img").all();
     const imgPromises = images.map((img) =>
       expect
-        .poll(
-          () => {
-            return img.evaluate((e) => (e as HTMLImageElement).complete);
-          },
-          {
-            message: "Wait for images in page preview to render.",
-            timeout: 1000,
-          }
-        )
+        .poll(() => img.evaluate((e) => (e as HTMLImageElement).complete), {
```

Comment:
instead of casting `e` to HTMLImageElement, does it work if you just set that as the type (i.e. `img.evaluate((e: HTMLImageElement) => e.complete)`)?